query/common/src/main/java/org/codice/ddf/admin/common/fields/base/scalar/LongField.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.common.fields.base.scalar;
+
+public class LongField extends BaseScalarField<Long> {
+
+  public static final String DEFAULT_LONG_FIELD_NAME = "long";
+
+  public LongField() {
+    this(DEFAULT_LONG_FIELD_NAME);
+  }
+
+  public LongField(String fieldName) {
+    super(fieldName, null, null, ScalarType.LONG);
+  }
+
+  public LongField(String fieldName, String description) {
```

Comment:
✏️ Same here, you should only be able to add description to custom field types, not the primitive type.